deal.II-toolchain/packages/dealii.package

```diff
@@ -17,6 +17,7 @@ INSTALL_PATH=${ORIG_INSTALL_PATH}/deal.II-${VERSION}
 
 CONFOPTS=" \
 -D CMAKE_BUILD_TYPE=DebugRelease \
+-D DEAL_II_EARLY_DEPRECATIONS:BOOL=OFF \
 -D DEAL_II_WITH_MPI:BOOL=ON \
```

Comment:
@tjhei as this is a developer option, maybe we should enable this feature if DEVELOPER_MODE=ON?

Reply:
or putting it to the "global" DEAL_II_CONFOPTS in candi.cfg would be another idea.